spec/support/pageflow/dummy/templates/database.yml

```diff
@@ -8,8 +8,10 @@ development:
 <% if ENV['PAGEFLOW_DB_HOST'].present? %>
   host: <%= ENV['PAGEFLOW_DB_HOST'] %>
   port: <%= ENV.fetch('PAGEFLOW_DB_PORT', '3306') %>
-<% else %>
+<% elsif File.socket?('/var/run/mysqld/mysqld.sock') %>
   socket: /var/run/mysqld/mysqld.sock
+<% elsif File.socket?('/tmp/mysql.sock') %>
+  socket: /tmp/mysql.sock
 <% end %>
```

Comment:
I'd suggest adding an `else` case here and failing with a helpful error message. Maybe we can use a YAML anchor to prevent duplication of these blocks since are starting to get more complex.